Cannot run SwiftWasm on macOS 10.15

I got this error after updating Tokamak to the latest SwiftWasm. Running macOS 10.15.5 (19F101).

```shellsession
$ carton dev --product TokamakDemo                                                                                                                                                                 
Polyfill integrity verified: /Users/jed/.carton/static/dev.js
Inferring basic settings...
- swift executable: /Users/jed/.carton/sdk/wasm-DEVELOPMENT-SNAPSHOT-2020-06-27-a/usr/bin/swift
Swift version 5.3-dev (LLVM a6315d2e8323dd3, Swift b80daa10576d040)
Target: x86_64-apple-darwin19.5.0
Parsing package manifest: /Users/jed/.carton/sdk/wasm-DEVELOPMENT-SNAPSHOT-2020-06-27-a/usr/bin/swift package dump-package
Error: Process failed with non-zero exit status and following error output:
/Users/jed/Documents/github-clones/Forks/Tokamak: error: manifest parse error(s):
Undefined symbols for architecture x86_64:
  "_$s18PackageDescription17SupportedPlatformV12MacOSVersionV6v10_15AEvau", referenced from:
      _main in Package-964a9c.o
ld: symbol(s) not found for architecture x86_64
<unknown>:0: error: link command failed with exit code 1 (use -v to see invocation)
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -79,7 +79,7 @@ jobs:
           name: macos-test-results
           path: ./swift-test-results.tar.gz
   macos_smoke_test:
-    name: Compile hello.swift on macOS
+    name: Run smoke tests on macOS
     runs-on: macos-latest
     needs: macos_build
     steps:
@@ -92,21 +92,29 @@ jobs:
         run: |
           set -x
           tar xf $(find . -name "swift-wasm-*.tar.gz" -type f)
-          TOOLCHAIN_PATH=$(find . -name "swift-wasm-*" -type d)
+          TOOLCHAIN_PATH=$(find "$PWD" -name "swift-wasm-*" -type d)
           echo 'print("Hello, world!")' > hello.swift
           $TOOLCHAIN_PATH/usr/bin/swiftc \
                 -target wasm32-unknown-wasi \
                 -sdk $TOOLCHAIN_PATH/usr/share/wasi-sysroot \
                 hello.swift -o hello.wasm && \
                 echo "Successfully linked hello.wasm"
-
+      - name: Test SwiftPM
+        shell: bash
+        run: |
+          set -x
+          TOOLCHAIN_PATH=$(find "$PWD" -name "swift-wasm-*" -type d)
+          mkdir test
+          cd test
+          $TOOLCHAIN_PATH/usr/bin/swift package init
+          $TOOLCHAIN_PATH/usr/bin/swift build --triple wasm32-unknown-wasi
       - name: Upload hello.wasm compiled with macOS package
         uses: actions/upload-artifact@v1
         with:
           name: macos-hello.wasm
           path: hello.wasm
   linux_smoke_test:
-    name: Compile hello.swift on Linux
+    name: Run smoke tests on Linux
     runs-on: ubuntu-18.04
     needs: linux_build
     steps:
@@ -119,14 +127,22 @@ jobs:
         run: |
           set -x
           tar xf $(find . -name "swift-wasm-*.tar.gz" -type f)
-          TOOLCHAIN_PATH=$(find . -name "swift-wasm-*" -type d)
+          TOOLCHAIN_PATH=$(find "$PWD" -name "swift-wasm-*" -type d)
           echo 'print("Hello, world!")' > hello.swift
           $TOOLCHAIN_PATH/usr/bin/swiftc \
                 -target wasm32-unknown-wasi \
                 -sdk $TOOLCHAIN_PATH/usr/share/wasi-sysroot \
                 hello.swift -o hello.wasm && \
                 echo "Successfully linked hello.wasm"
-
+      - name: Test SwiftPM
+        shell: bash
+        run: |
+          set -x
+          TOOLCHAIN_PATH=$(find "$PWD" -name "swift-wasm-*" -type d)
+          mkdir test
+          cd test
+          $TOOLCHAIN_PATH/usr/bin/swift package init
+          $TOOLCHAIN_PATH/usr/bin/swift build --triple wasm32-unknown-wasi
       - name: Upload hello.wasm compiled with Linux package
         uses: actions/upload-artifact@v1
         with:

